src/test/java/DirListHtmlTest.java

```diff
@@ -0,0 +1,48 @@
+import com.td.HttpServer.*;
+
+public class DirListHtmlTest extends junit.framework.TestCase {
+
+  DirListHtml dirListHtml;
+  String pathRoot;
+  String path;
+  String[] mockFiles;
+  String builtHtmlPage;
+  String builtHtmlPageRootDirForPath;
+
+  protected void setUp() {
+    dirListHtml = new DirListHtml();
+    pathRoot = "/";
+    path = "myPath";
+    mockFiles = new String[] { "File01.txt", "directory01" };
+    builtHtmlPage = new String(dirListHtml.buildHtmlPage(path, mockFiles));
+    builtHtmlPageRootDirForPath = new String(dirListHtml.buildHtmlPage(pathRoot, mockFiles));
```

Comment:
I would probably create this and `pathRoot` directly in the one test that uses them. Having the variables defined here here suggests that multiple tests use them (like `builtHtmlPage`).